### PR TITLE
Sql Binding: Add "Check out pane for more details" when nuget package download fails

### PR DESCRIPTION
**Pull request by lucyzhang929**
This PR fixes https://github.com/microsoft/azuredatastudio/issues/17580

Screen Shot 2021-11-16 at 9 23 50 AM

### CODE REVIEW

File: extensions/sql-database-projects/src/tools/packageHelper.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export class PackageHelper {
 				});
 			}
 		} catch (e) {
-			void vscode.window.showErrorMessage(e.message);
+			void vscode.window.showErrorMessage(e.message, constants.checkoutOutputMessage);
```
[the remark this answers is not in the document]
Comment (lucyzhang929) [reply extensions/sql-database-projects/src/tools/packageHelper.ts:75]: Yep I can add that

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/sql-database-projects/src/tools/packageHelper.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class PackageHelper {
 				});
 			}
 		} catch (e) {
-			void vscode.window.showErrorMessage(e.message);
+			void vscode.window.showErrorMessage(constants.addSqlBindingPackageError, constants.checkoutOutputMessage);
```
Comment (kisantia) [inline extensions/sql-database-projects/src/tools/packageHelper.ts:75]: does this need to wait for the result from `showErrorMessage()` to show the output channel? Like in https://github.com/microsoft/azuredatastudio/blob/b8ea493f8cca96d608a560834a2779ea9aae1f5d/extensions/sql-database-projects/src/common/utils.ts#L519-L522

Comment (kisantia) [reply extensions/sql-database-projects/src/tools/packageHelper.ts:75]: oh actually maybe you can reuse `showErrorMessageWithOutputChannel()` here

Comment (lucyzhang929) [reply extensions/sql-database-projects/src/tools/packageHelper.ts]: ah yes thanks for catching that - added the fix

**Review by kisantia — APPROVED** [review]

**Pull request merged**